components/contact-us-form.tsx

```diff
@@ -0,0 +1,193 @@
+
+                        <span>marketing@luminate.com</span>
+
+                    </div>
+
+                    <div className="flex items-center">
+
+                        <img src="/images/icons/phone-flip.svg" alt="Email" className="mr-4" style={{ width: '1rem', height: '1rem' }} />
+
+                        <span>+92 3361000001</span>
+
+                    </div>
+
+                    <div className="flex items-center">
+
+                        <img src="/images/icons/location-pin.svg" alt="Email" className="mr-4" style={{ width: '1rem', height: '1rem' }} />
+
+                        <span>Islamabad, Pakistan</span>
+
+                    </div>
+
+                </div>
+
+            </div>
+
+            <div className="md:w-1/2">
+
+                <h2 className="animate-[gradient_6s_linear_infinite] bg-[linear-gradient(to_right,theme(colors.gray.200),theme(colors.forest.200),theme(colors.gray.50),theme(colors.forest.300),theme(colors.gray.200))] bg-[length:200%_auto] bg-clip-text pb-4 font-menda text-3xl font-medium text-transparent md:text-3.5xl">
+
+                    Let&apos;s Collaborate to Create Something Extraordinary
+
+                </h2>
+
+                <form onSubmit={handleSubmit(onSubmit)} style={{ display: 'flex', flexDirection: 'column' }}>
+                    <div style={{ marginBottom: '15px' }}>
+                        <label style={{ color: 'white', marginBottom: '5px' }}>Name</label>
+                        <input
+                            type="text"
+                            placeholder="Name"
+                            {...register("name")}
+                            style={{
+                                width: '100%',
+                                padding: '10px',
+                                border: 'none',
+                                borderRadius: '5px',
+                                backgroundColor: 'transparent',
+                                color: 'white',
+                                boxShadow: '0 -3px 5px rgba(0, 162, 112, 0.5), 0 3px 5px rgba(0, 162, 112, 0.5)',
+                            }}
+                        />
+                        {errors.name && <p style={{ color: 'red' }}>{errors.name.message}</p>}
+                    </div>
+
+                    <div style={{ marginBottom: '15px' }}>
+                        <label style={{ color: 'white', marginBottom: '5px' }}>Email</label>
+                        <input
+                            type="email"
+                            placeholder="Email"
+                            {...register("email")}
+                            style={{
+                                width: '100%',
+                                padding: '10px',
+                                border: 'none',
+                                borderRadius: '5px',
+                                backgroundColor: 'transparent',
+                                color: 'white',

```

Comment:
Again, same comment as above. why are inline styles being used. If there is a very necessary thing that cannot be done in tailwind then only use inline, else follow the practices already defined in the codebase